### ISSUE TITLE
How can I generate the Html report 

### ISSUE THREAD

**Issue by gonmedtara** [sonarwhal/sonarwhal#728]
Hello **Sonarwhal** team, how can I get the `html report`? what `formatter `I can choose? thank you.

**Comment by alrra** [comment]
@gonmedtara The CLI doesn't yet have a `formatter` that outputs HTML.

We started working [on it](https://github.com/sonarwhal/sonarwhal/pull/411), but at that point we decided to finish the [online version](https://sonarwhal.com/scanner/) first and only after resume / restart work on it).

See also: #722.

---

Closing in favor of #284.
